src/main/java/org/jenkinsci/plugins/displayurlapi/DisplayURLContext.java

```diff
@@ -88,7 +88,7 @@ private DisplayURLContext(DisplayURLContext parent) {
      */
     private void guessPlugin() {
         StackTraceElement[] stack = (new Throwable()).getStackTrace();
-        PluginManager manager = Jenkins.getActiveInstance().getPluginManager();
+        PluginManager manager = Jenkins.getInstance().getPluginManager();
```

Comment:
Is `Jenkins.get()` already available in the used Jenkins version? Then this should be used.

Reply:
Sadly it is not 😢